xml/System.Net.Http/HttpRequestException.xml

```diff
@@ -141,5 +141,92 @@
+      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception, statusCode As Nullable(Of System.Net.HttpStatusCode)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; HttpRequestException(System::String ^ message, Exception ^ inner, Nullable<System::Net::HttpStatusCode> statusCode);" />
+      <MemberSignature Language="F#" Value="new System.Net.Http.HttpRequestException : string * Exception * Nullable<System.Net.HttpStatusCode> -&gt; System.Net.Http.HttpRequestException" Usage="new System.Net.Http.HttpRequestException (message, inner, statusCode)" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Net.Http</AssemblyName>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <AssemblyInfo>
+        <AssemblyName>netstandard</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>2.1.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="message" Type="System.String" />
+        <Parameter Name="inner" Type="System.Exception" />
+      </Parameters>
+      <Docs>
+        <param name="message">A message that describes the current exception.</param>
+        <param name="inner">The inner exception.</param>
+        <param name="statusCode">The HTTP status code.</param>
+        <summary>Initializes a new instance of the <see cref="HttpRequestException" /> class with a specific message that describes the current exception, an inner exception, and an HTTP status code.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="StatusCode">
+      <MemberSignature Language="C#" Value="public System.Net.HttpStatusCode? StatusCode { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.Nullable<valuetype System.Net.HttpStatusCode> StatusCode" />
+      <MemberSignature Language="DocId" Value="P:System.Nullable{System.Net.HttpStatusCode}" />
+      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property StatusCode As Nullable(Of System.Net.HttpStatusCode)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property Nullable<System::Net::HttpStatusCode> ^ StatusCode { Nullable<System::Net::HttpStatusCode> ^ get(); };" />
+      <MemberSignature Language="F#" Value="member this.StatusCode : Nullable<System.Net.HttpStatusCode>" Usage="System.Net.Http.HttpRequestException.StatusCode" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Net.Http</AssemblyName>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <AssemblyInfo>
+        <AssemblyName>netstandard</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>2.1.0.0</AssemblyVersion>
```

Comment:
We should remove the .NET Standard versions - they are not applicable. The property is not part of them.